src/transformStateWithClones.js

```diff
@@ -66,6 +66,32 @@
  */
 function transformStateWithClones(state, transforms) {
   // write code here
+
+  const arrayOfStates = [];
```

Comment:
do not use array word
```suggestion
  const states = [];
```